proposals/user-scripts-api.md

```diff
@@ -52,9 +52,12 @@ User scripting related features will be exposed in a new API namespace, tentativ
 #### Types
 
 ```
+// See RegisteredUserScript validation section, below.
 dictionary RegisteredUserScript {
   boolean? allFrames;
-  ScriptSource[] js;
+  // js is required in userScripts.register(), optional in userScripts.update().
+  // When specified, must be a non-empty array.
```

Comment:
Side note: Chrome currently requires `js` to be a non-empty array, Firefox currently accepts an empty array.